docs/reference/filters.md

```diff
@@ -976,6 +976,23 @@ Example:
 * -> blockContent("Malicious Content") -> "http://example.com";
 ```
 
+### blockHexContent
```

Comment:
Lets maybe use `Hex` suffix, i.e. `blockContentHex` similar to `repeatContentHex` and `wrapContentHex`

Reply:
done